monitor/sortedOracles.go

```diff
@@ -87,6 +89,50 @@ func (p sortedOraclesProcessor) ObserveState(opts *bind.CallOpts, stableTokenAdd
 	return nil
 }
 
+func (p sortedOraclesProcessor) ObserveMetric(opts *bind.CallOpts, stableTokenAddress common.Address) error {
+
+	isOldestReportExpired, _, err := p.sortedOracles.IsOldestReportExpired(opts, stableTokenAddress)
+	if err != nil {
+		return err
+	}
+	metrics.SortedOraclesIsOldestReportExpired.Set(utils.BoolToMetric(isOldestReportExpired))
+
+	numRates, err := p.sortedOracles.NumRates(opts, stableTokenAddress)
+	if err != nil {
+		return err
+	}
+	metrics.SortedOraclesNumRates.Set(float64(numRates.Uint64()))
+
+	medianRateNumerator, medianRateDenominator, err := p.sortedOracles.MedianRate(opts, stableTokenAddress)
+	if err != nil {
+		return err
+	}
+	medianRate := big.NewFloat(0)
+
+	if medianRateDenominator.Cmp(big.NewInt(0)) != 0 {
+		retN := new(big.Float).SetInt(medianRateNumerator)
+		retD := new(big.Float).SetInt(medianRateDenominator)
+		medianRate = new(big.Float).Quo(retN, retD)
+	}
+	medianRateMetric, _ := medianRate.Float64()
+	metrics.SortedOraclesMedianRate.Set(medianRateMetric)
+
+	_, rateValues, _, err := p.sortedOracles.GetRates(opts, stableTokenAddress)
+	if err != nil {
+		return err
+	}
+	metrics.SortedOraclesMeanRate.Set(utils.Mean(rateValues))
```

Comment:
Is `mean(x)` enough or should we export `x/mean(x)`? the latter implies exporting an array of normalized values